server/src/routes/api/v1/reviewsRouter.js

```diff
@@ -0,0 +1,16 @@
+import express from "express";
+import { Review } from "../../../models/index.js";
+
+const reviewsRouter = new express.Router();
+
+reviewsRouter.delete("/:id", async (req, res) => {
+  const id = req.params.id;
+  try {
+    await Review.query().deleteById(id);
```

Comment:
there is handling on the front end to make sure only the user who left the review cans ee the delete button and delete the review, but we need to have that similar validation check in the back end as well that only if the current user is the same as the user associated with the review